api.raml

```diff
@@ -371,6 +371,21 @@ securitySchemes:
               application/json:
                 type: lib.ScatterplotPostResponse
 
+  /differentialabundance/visualizations:
+    displayName: Visualizations for interrogating differentially abundant taxa, genes, or pathways
```

Comment:
Should just say taxa for now?

Reply:
The EC numbers are enzymes so i was hoping to dump them into the "genes" umbrella